docs/csharp/misc/cs1951.md

```diff
@@ -15,13 +15,13 @@ author: "BillWagner"
 ms.author: "wiwagn"
 ---
 # Compiler Error CS1951
-An expression tree lambda may not contain an out or ref parameter.  
+An expression tree lambda may not contain an in, out or ref parameter.  
```

Comment:
Are these intended to be literal error messages? If so, it's OK if the punctuation is inconsistent, but otherwise they should be consistent.

Reply:
I'll be consistent. (I've filed a bug that the compiler only reports ref and out) See dotnet/roslyn#25174